test/demarcheSimplifiees/autoVerifyPsychologist.spec.js

```diff
@@ -0,0 +1,74 @@
+const rewire = require('rewire');
+const sinon = require('sinon');
+
+const demarchesSimplifiees = rewire('../../services/demarchesSimplifiees');
+const graphql = rewire('../../utils/graphql');
+
+describe('autoVerifyPsychologist', () => {
+  let executeMutationStub;
+  let unsets = [];
+
+  beforeEach(() => {
+    unsets = [];
+    executeMutationStub = sinon.stub();
+    unsets.push(graphql.__set__('executeMutation', executeMutationStub));
+    unsets.push(demarchesSimplifiees.__set__('graphql', graphql));
+  });
+
+  afterEach((done) => {
+    unsets.forEach((unset) => unset());
+    done();
+  });
+
+  it('Should execute 4 mutations on DS', async () => {
+    await demarchesSimplifiees.autoVerifyPsychologist();
+
+    sinon.assert.callCount(executeMutationStub, 4);
+
+    const DOSSIER_ID_VALID = 'RG9zc2llci00NzI5ODc3';
+    const DOSSIER_ID_INVALID = 'RG9zc2llci00NzI5ODcw';
+    const INSTRUCTOR_ID = 'SW5zdHJ1Y3RldXItNDg5OTM='; // TODO: update once DS test account created
```

Comment:
use config info (in .env.ci)

Reply:
I wanted to avoid it in the test to ensure the correct value was in .env.ci but it's a bit of a pain so I'll use config info :+1: